### PR TITLE
Integration

### PR DESCRIPTION
**Pull request by Sax593**

### CODE REVIEW

**Review by guenk — APPROVED** [review]
perfect 10/10

File: package.json [hunk 1]
```diff
@@ -19,5 +19,7 @@
     "concurrently": "^7.0.0",
     "husky": "^7.0.0"
   },
-  "dependencies": {}
-}
\ No newline at end of file
+  "dependencies": {
+    "prop-types": "^15.8.1"
```
Comment (LoicBrassart) [inline package.json]: Should be in the frontend :(

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Saw almost no Evil
... Haven't I reviewed a **very** similar code in your other PR this morning ? Please don't duplicate your components on several branches, it makes my job harder -_-

**Review by LoicBrassart — APPROVED** [review]
No more Evil!

**Pull request merged**